Formula/xmrig.rb

```diff
@@ -31,21 +22,5 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{bin}/xmrig -V")
-    test_server="donotexist.localhost:65535"
-    timeout=10
-    begin
-      read, write = IO.pipe
-      pid = fork do
-        exec "#{bin}/xmrig", "--no-color", "--max-cpu-usage=1", "--print-time=1",
-             "--threads=1", "--retries=1", "--url=#{test_server}", :out => write
-      end
-      start_time=Time.now
-      loop do
-        assert (Time.now - start_time <= timeout), "No server connect after timeout"
-        break if read.gets.include? "\] \[#{test_server}\] DNS error: \"unknown node or service\""
-      end
-    ensure
-      Process.kill("SIGINT", pid)
-    end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I tried to leave that block untouched, but then the test on Jenkins failed after 5 minutes. I am now restoring that block and force pushed, hopefully the test can complete this time.